utils/notionTypeParser/imageParser.jsx

```diff
@@ -1,24 +1,23 @@
 import React from 'react'
-import { api } from '../../next.config'
 import styles from '../../components/ManualPage/Template.module.css'
+import { API_HOST } from '../../consts/endpoints'
 
 const getImage = (imageObj) => {
+    const image = (
+        <img
+            className={styles.templateImage}
+            src={`${API_HOST}/static/${imageObj.content.image_name}`}
+            alt=""
```

Comment:
Maybe get another field from the backend somehow with information about the content (improve a11y)

Reply:
I think we don't have the server to recognize all the images we work with yet